recipes/rb-json/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "json" %}
+{% set version = "2.2.0" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+  sha256: 9dd1437156773f72c096058ec837faac1b00077121a3fd574e68f895ea3aa96b
+
+build:
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies {{ name }}-{{ version }}.gem
+    - make -C {{ PREFIX }}/lib/ruby/gems/{{ ruby }}.0/gems/{{ name }}-{{ version }}/ext/{{ name }} clean
+  skip: {{ win }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - ruby
+    - openssl
```

Comment:
It looks like `openssl` is not a requirement from the library according to the [build logs][linuxlogs] either. Please try building without it if you haven't tried it yet, o.w. just let me know.
<details>
  <summary>build logs snippet: lines [640,641]</summary>

```
WARNING (rb-json): run-exports library package conda-forge::openssl-1.1.1a-h14c3975_1001 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
```
</details>

```suggestion
```

[linuxlogs]: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=68987&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c&taskId=bef775d0-a1a2-55a3-9721-17147029baf0&lineStart=640&lineEnd=641&colStart=1&colEnd=1